tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/varref/ErrorVariableReferenceTest.java

```diff
@@ -360,7 +360,24 @@ public void testErrorDetailBindingNegative() {
                 "invalid field binding pattern; can only bind required fields", 53, 14);
         BAssertUtil.validateError(resultNegative, i++,
                 "invalid field binding pattern; can only bind required fields", 54, 18);
-
+        BAssertUtil.validateError(resultNegative, i++, "cannot use a wildcard binding pattern to " +
+                "bind the cause: a wildcard binding pattern can be used only with a value that belong to " +
+                "type 'any'", 74, 25);
```

Comment:
Since the error message is fixed, let's extract it out to a constant in this file. Helps if we have to change it at some point too.